docs/source/development/releasing.rst

```diff
@@ -384,4 +378,17 @@ Be sure to go through on the following checklist:
       # dev/release/post-07-bump-versions.sh ../arrow 0.1.0 0.2.0
       dev/release/post-07-bump-versions.sh <arrow-dir> <version> <next_version>
 
+.. dropdown:: Publish release blog post
+   :class-title: sd-fs-5
+   :class-container: sd-shadow-md
+
+   Run the script to generate the blog post outline, then fill out the
+   outline and create a PR on `apache/arrow-site
+   <https://github.com/apache/arrow-site>`_.
+
+   .. code-block:: Bash
+
+      # dev/release/post-08-website.sh ../arrow-site 0.0.0 0.1.0
+      dev/release/post-08-website.sh <arrow-site-dir> <prev_version> <version>
```

Comment:
Could you use `09` instead of `08`?
Because I used `08` in #337.

Reply:
Rebased + updated the filename.

Also changed it to extract API version from the header file.